exercises/anagram/anagram_test.rb

```diff
@@ -3,70 +3,156 @@
 require 'minitest/autorun'
 require_relative 'anagram'
 
+# Test data version:
+# 3dfde31
 class AnagramTest < Minitest::Test
   def test_no_matches
+    # skip
     detector = Anagram.new('diaper')
-    assert_equal [], detector.match(%w(hello world zombies pants))
+    anagrams = detector.match(["hello", "world", "zombies", "pants"])
+    assert_equal [].sort, anagrams.sort
   end
 
-  def test_detect_simple_anagram
+  def test_detects_simple_anagram
     skip
     detector = Anagram.new('ant')
-    anagrams = detector.match(%w(tan stand at))
-    assert_equal ['tan'], anagrams
+    anagrams = detector.match(["tan", "stand", "at"])
+    assert_equal ["tan"].sort, anagrams.sort
```

Comment:
(This is an 'in an ideal world' type issue please don't think that you **have** to do this.)

Based on the principle that we want to make the test cases as nice as possible for people and would prefer generator complexity over test case complexity. 
It could be nice to omit the `.sort` when there is only one element, and pre-sort the 'expected' arrays that have multiple elements.